lib/ranked-model.rb

```diff
@@ -65,6 +65,16 @@ def ranks *args
         ranker.with(self).relative_rank
       end
 
+      define_method "#{ranker.name}_insert_after!" do |previous|
+        if previous == '-1'
```

Comment:
Setting the assumption, when an object has to be moved to the top of the list, set `previous` explicitly as 1.

Reply:
I wanted to prefix the keyword `previous` so that it is explicit that we are sending the previous record in the list and insertion will happen only after the previous record.

Reply:
I think the name of the method implies the action, so the variable should name what we're passing in. e.g. `def insert_after!(ranked_item)` as a method definition.

Reply:
`def insert_after!(previous_record)` vs `def insert_after!(record)` => The second looks 👌  

I think I would name it as `record` Let me know what you think.

Reply:
I think that sounds good :) The `_previous` isn't required because it is implied by `_after`.